QST-1009/index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+    <script src="script.js" defer></script>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Barlow:wght@300;400;500&family=Poppins:wght@200;300;400;600;700&display=swap" rel="stylesheet">
+    <title>Product Listing Page with API</title>
+</head>
+<body>
+
+<div class="page-container">
+    <nav>
+        <div class="navbar">
+            <ul>
+                <li class="left-navbar-items"> <img src="logo-color.svg" id="logo"> </li>
+                <li class="left-navbar-items">WOMEN</li>
+                <li class="left-navbar-items">MEN</li>
+                <li class="left-navbar-items">KIDS</li>
+                <li class="left-navbar-items">SPORTSWEAR</li>
+                <li class="left-navbar-items">BRANDS</li>
+                <li class="left-navbar-items">SWIMWEAR</li>
+                <li class="left-navbar-items">RUNNING</li>
+                <li class="left-navbar-items">SALE</li>
+                <li class="navbar-space"></li>
+                <li class="right-navbar-items"><img src="map-pin-line.svg" height="20px"></li>
+                <li class="right-navbar-items"><img src="user-3-line.svg" height="20px"></li>
+                <li class="right-navbar-items"><img src="shopping-bag-line.svg" height="20px"></li>
+            </ul>
+        </div>
+        <div class="search-bar">
+            
+                <h4>What are you looking for?</h2>
+            
+             
+            <span> <input type="text" id="search-box"> <i id="search-icon"> <img src="search-line.svg" height="20px"> </i> </span>
```

Comment:
place all images within a folder called assets and update src path instead of leaving it in root.